### ISSUE TITLE
Question about mini cart

### ISSUE THREAD

**Issue by rolzav** [bootscore/bootscore#263]
Hi, Basti & Justin & other contributors and users!

I would like to know, what could be a reason for infinity loading mini cart (while adding a product). After page reload, it shows in a badge +1 product (in a nav bar), but when i`m trying to add a product (in a single product page) minicart is going round and round.

First time this problem. For previous webs everything was good so far.

Thanks and have a productive day!

**Comment by rolzav** [comment]
Hopefuly I will publish a demo page on some server after few hours, then Yes.  B.t.w if Im using latest Parent theme and bootcommerce and woo version 6.9.0, then everything should be fine, right?

**Comment by crftwrk** [comment]
> B.t.w if Im using latest Parent theme and bootcommerce and woo version 6.9.0, then everything should be fine, right?

Yes, this should be fine. I'm not sure if it's a server respond or an JS error, that's why a link would be very helpful.

**Comment by rolzav** [comment]
http://metropole.netcaremasters.com/product/product-one/

Look like it`s working fine on host

**Comment by rolzav** [comment]
I found a problem on localhost. Look like its all fine with the theme of course!

**Comment by rolzav** [comment]
HI, again. I found a problem on localhost.

but only error that comes up is this:
__
Your theme (bootCommerce Child) contains outdated copies of some WooCommerce template files. These files may need updating to ensure they are compatible with the current version of WooCommerce. Suggestions to fix this:

--


And in woocommerce>status this error:
bootscore/woocommerce/myaccount/form-login.php version 4.1.0 is out of date. The core version is 6.0.0,


everything else looks great!

**Comment by crftwrk** [comment]
Your site looks good. Think your local server did not completed the ajax request, that's why you only have seen the cart-loader.

> Your theme (bootCommerce Child) contains outdated copies of some WooCommerce template files. 

bootcommerce-child does not include any WC template file (exept you copied some of them to yours). All templates are in the main-theme.

`form-login.php` has been already updated here https://github.com/bootscore/bootscore/pull/255. Download latest 5.2.1.1 release https://github.com/bootscore/bootscore/releases/download/v5.2.1.1/bootscore-main.zip and update via the theme-uploader. No breaking changes since 5.2.0.0.

Solved?

**Comment by rolzav** [comment]
Yes!

**Comment by crftwrk** [comment]
👍
